### DISCUSSION TITLE
Rogue comma

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#478] [Get Help] [answered]
I've noticed in a couple of places that there is an extra space added in front of a comma, usually after a link. This makes the text wrap weird: 

<img width="799" alt="Screen Shot 2022-09-07 at 4 41 45 PM" src="https://user-images.githubusercontent.com/74433007/188988035-3da8c003-a37b-4402-bfeb-35dca97998ae.png">

There is not an extra space in my markdown file so I'm not sure how to fix it. Any advice?

<img width="1307" alt="Screen Shot 2022-09-07 at 4 42 50 PM" src="https://user-images.githubusercontent.com/74433007/188988337-cdf017c2-9824-435c-8356-f29580828d36.png">

**Comment by geealbers** [comment, marked as answer]
@mmpadilla15 I'm not sure if it will help, but can you make sure there are not trailing spaces or line breaks in the `layouts/_default/_markup/render-link.html` file that you added to get external links working?

**Reply by mmpadilla15** [reply]
I tabbed through it to see if there were any invisible spaces, but I wasn't able to find any. This is what it looks like:

![Screen Shot 2022-09-08 at 9 57 15 AM](https://user-images.githubusercontent.com/74433007/189156085-3b49a1e4-7213-4df2-9d68-7290934af4d4.png)

**Reply by geealbers** [reply]
@mmpadilla15, thanks for including the screenshot. We want to delete that empty second line. If you put your cursor on line 2, and delete so that it moves to the end of line 1 (right after the `</a>`, and then save the file, that should do it. If line 2 shows back up after saving, there's a setting in your text editor that needs to be changed. For that, if you're using Atom, you go to Preferences, look for Packages and search "whitespace" and that package will have some checkboxes that should do it.

**Reply by mmpadilla15** [reply]
@geealbers That worked! Thank you!
